app/components/standup_meeting_group/check_in_status_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if @standup_meeting.present? && @standup_meeting.completed? %>
```

Comment:
I might like condensing this `if` check down to `if @standup_meeting&.completed?`, but that's minor and not something you have to change.

Reply:
I like your suggestion better!